### PR TITLE
Kubernetes / Helm cleanups

### PR DESCRIPTION
**Pull request by cmgrote**
- Removes root-level execution from `egeria-configure` container image
- Separates IBM IGC proxy more cleanly (for some eventual state where it may make sense to put as a separate tenant in a single OMAG Server Platform: not yet possible due to hostname requirements, but may change with CoreDNS inclusion in k8s v1.13+, or OpenShift (which already uses CoreDNS over kube-dns))
- Fixes a troublesome readiness check during the initialisation of the IGC piece: now ensures IGC REST API is fully-ready before proceeding with the initialisation (previously could have proceeded thinking it was ready, but would have resulted in a 500 error)
- Adds setup of an empty `LOADER_PATH` directory in the Egeria pod, for eventual inclusion of the Apache Atlas proxy connector (and longer-term potentially the IGC one, per above)
- Adds new `apache-atlas` image that uses a vanilla build of Apache Atlas (no fork, patches, etc), for baseline development of an Apache Atlas proxy connector; also ensuring it runs as a non-root user
- Adds an option (disabled by default) to automatically run the Conformance Test Suite (CTS) against any repository that starts up as part of the Helm-orchestrated cohort

### CODE REVIEW

**Comment by cmgrote** [comment]
Didn't realise this hadn't been merged yet -- updating the original description with further changes / fixes...

**Comment by planetf1** [comment]
Not merged yet as was challenged on testing as getting some atlas issues, and I know 2 groups using the chart & using Atlas UI to browse
will investigate more today.

File: open-metadata-resources/open-metadata-deployment/docker/configure/Dockerfile [hunk 1]
```diff
@@ -15,6 +15,12 @@ LABEL org.label-schema.docker.params = "SERVICE=the name of the k8s service to c
 
 # Install utilities we will use to configure containers
 RUN apk --no-cache add bash curl git jq openldap-clients postgresql-client mariadb-client && apk --no-cache update && apk --no-cache upgrade
-COPY dist/check-availability.sh /check-availability.sh
+
+RUN groupadd -r config -g 1000 && useradd --no-log-init -r -g config -u 1000 -d /opt/config config
```
Comment (planetf1) [inline open-metadata-resources/open-metadata-deployment/docker/configure/Dockerfile]: groupadd not present, resulting in a build failure
```
[INFO] Step 12/16 : RUN groupadd -r config -g 1000 && useradd --no-log-init -r -g config -u 1000 -d /opt/config config
[INFO]
[INFO]  ---> Running in 7f69ba971f00
[INFO] /bin/sh: groupadd: not found
[INFO]
[ERROR] The command '/bin/sh -c groupadd -r config -g 1000 && useradd --no-log-init -r -g config -u 1000 -d /opt/config config' returned a non-zero code: 127
[WARNING] An attempt failed, will retry 1 more times
org.apache.maven.plugin.MojoExecutionException: Could not build image
```

Comment (planetf1) [reply open-metadata-resources/open-metadata-deployment/docker/configure/Dockerfile]: The fix here would be to add the shadow package (ie apk add shadow -- or more likely add to prior add.... I notice the atlas image already ends up including groupadd). Or could use addgroup. I'd go with shadow.

Comment (planetf1) [reply open-metadata-resources/open-metadata-deployment/docker/configure/Dockerfile]: will apply that change locally & do a full rebuild to test before merge (assuming above fix)

**Comment by planetf1** [comment]
will retest with latest changes.

**Comment by planetf1** [comment]
Tested ok (basic cloud config). Merging

**Pull request merged**